pkg/controller/ibmblockcsi/syncer/csi_controller.go

```diff
@@ -148,7 +150,7 @@ func (s *csiControllerSyncer) ensureContainersSpec() []corev1.Container {
 }
 
 func ensureDefaultResources() corev1.ResourceRequirements {
-	return ensureResources("50m", "100m", "50Mi", "100Mi")
+	return ensureResources("20m", "40m", "20Mi", "40Mi")
```

Comment:
r u sure only 40Mi limit?

Reply:
yes, from CSI-643, all the sidecars only need no more than 23m and 20Mi for 3 pods and 300 volumes